whatlies/embeddingset.py

```diff
@@ -580,6 +592,94 @@ def plot_correlation(self, metric=None):
         # Rotate the tick labels and set their alignment.
         plt.setp(ax.get_xticklabels(), rotation=90, ha="right", rotation_mode="anchor")
 
+    def plot_similarity(self, metric="cosine", norm=False):
+        """
+        Make a similarity plot. Shows you the similarity between all the word embeddings in the set.
+
+        Arguments:
+            metric: `cosine` or `correlation`
```

Comment:
I think these should be wrapped in single or double quotation to make it clear they are strings, i.e. `'cosine'` or `'correlation'`.